image_writers_guide/guidelines.adoc

```diff
@@ -9,4 +9,135 @@
 
 toc::[]
 
-== Overview
\ No newline at end of file
+== Overview
+When creating Docker images to run on OpenShift there are a number of best practices to consider in order to ensure consumers of those images have a good experience.  Since images are intended to be immutable and used as is, it is the image author's responsibility to ensure the image contains any artifacts that would typically be needed by a user of the image and that the image can be configured by the user.  Here are some general comments around things to consider.
+
+== General Docker Best Practices
+The following are some guidelines that apply to Docker image creation in general, independent of whether the images are used on OpenShift v3.
```

Comment:
@smarterclayton asked me to move volumes up into the general section.  i'd rather not split it into two parts because the "general" vs "openshift" categories aren't really that important compared to the importance of keeping relevant information lumped together.   For now i'm going to leave this here, but perhaps as the openshift volume docs evolve we can change it up.